src/main/play-doc/developer/SignalingApplicationState.md

```diff
@@ -39,3 +39,9 @@ StatusService.signalStartedOrExit()
 ```
 
 After reading the following sections, you may also wish to refer to [the reference documentation for conductr-bundle-lib](https://github.com/typesafehub/conductr-bundle-lib#typesafe-conductr-bundle-library).
+
+## Application Exit Codes
+
+ConductR uses your application's exit code to determine whether it should be restarted in order to reform the cluster. Following POSIX conventions, applications with non-zero exit codes are considered failed and will be restarted, while applications exiting with exit code 0 will be viewed as having exited normally, and thus ConductR will take no action.
+
+This is particularly important for Akka applications using [Split Brain Resolver](http://doc.akka.io/docs/akka/rp-15v09p01/scala/split-brain-resolver.html) with the [registerOnMemberRemoved](http://doc.akka.io/docs/akka/rp-15v09p01/scala/split-brain-resolver.html#Strategies) callback to shut down the ActorSystem after a split brain downing decision has been made. A non-zero return code must be issued to ensure that ConductR is instructed to recover the cluster to its previous state.
```

Comment:
How about this for the time being: I'll add a code sample here based on the current registerOnMemberRemoved. It will return -1 for all cases, and diverge from the sample code in the SBR docs. Handling the graceful shut down case (where ConductR does not restart after leaving) won't be covered yet, but I figure ConductR's ability to restart a downed node takes precedence.

@patriknw, can you create an issue in the Akka repo for these two new methods if you think they're the best way to go for a timely solution - can always improve on it later. Graceful shutdown is something we seemed to get asked about a lot, so it'd be good to have a solid story that works with ConductR and SBR.

We can update the ConductR and SBR docs with the new callbacks once they've been added.